lib/buffer.js

```diff
@@ -133,13 +133,20 @@ Buffer.from = function(value, encodingOrOffset, length) {
 Object.setPrototypeOf(Buffer.prototype, Uint8Array.prototype);
 Object.setPrototypeOf(Buffer, Uint8Array);
 
+function assertSize(size) {
+  if (typeof size !== 'number') {
+    const err = new TypeError('"size" argument must be a number');
+    Error.captureStackTrace(err, assertSize);
```

Comment:
I'm confused. Is the only reason for this to remove `assertSize()` from the stack?